sdk/purview/azure-analytics-purview-sharing/CHANGELOG.md

```diff
@@ -2,14 +2,10 @@
 
 ## 1.0.0-beta.2 (Unreleased)
 
-### Features Added
-
-### Breaking Changes
-
-### Bugs Fixed
-
 ### Other Changes
 
+- Reverted internal sync stack changes until enhancement is released.
```

Comment:
can you also update the swagger readme to remove the enable-sync-stack entry?

Reply:
Done.